styles.css

```diff
@@ -13,707 +13,801 @@ body {
   max-width: 100%;
 }
 
-@media (min-width: 375px) {
-  .container {
-    display: flex;
-    background: #ffff;
-    border-radius: 0.05vw 0.05vw 0.05vw 30vw;
-    flex-direction: column;
-    background-image: url("images/header-shapes-mobile.png");
-    background-repeat: no-repeat;
-    background-size: 100%;
-    background-position: 100%;
-    position: absolute;
-    overflow-x: hidden;
-    width: 100%;
-    min-height: 100vh;
-  }
+/* @media (min-width: 375px) { */
+.container {
+  display: flex;
+  background: #ffff;
+  border-radius: 0.05vw 0.05vw 0.05vw 30vw;
+  flex-direction: column;
+  background-image: url("images/header-shapes-mobile.png");
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-position: 100%;
+  position: absolute;
+  overflow-x: hidden;
+  width: 100%;
+  min-height: 100vh;
+}
 
-  a:link {
-    text-decoration: none;
-  }
+a:link {
+  text-decoration: none;
+}
 
-  ul,
-  li {
-    list-style: none;
-  }
+ul,
+li {
+  list-style: none;
+}
 
-  .topnav {
-    align-items: center;
-    padding: 0.2em;
-  }
+.topnav {
+  align-items: center;
+  padding: 0.2em;
+}
 
-  .navi {
-    /* border: 2px solid; */
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    align-items: center;
-    font-size: 18px;
-    padding: 0.5rem;
-  }
+.navi {
+  /* border: 2px solid; */
```

Comment:
- [ ]  Kindly remove all the non descriptive commented code in this file for a cleaner codebase